app/Config/Routes.php

```diff
@@ -14,3 +16,6 @@
 $routes->post('account/login', [Account::class, 'login']);
 $routes->post('account/logout', [Account::class, 'logout']);
 $routes->get('account', [Account::class, 'get'], ['filter' => AuthFilter::class]);
+$routes->get('images/(:segment)', [Image::class, 'get']);
```

Comment:
I don't get what that Route is for.
I thought I geht the route as a string and just drop it into a image tag. 
Or do I miss something?

Reply:
You're correct, you get the image link as part of the JSON response, e.g. "images/coder2k.jpg". For your image tag, you then use "api/images/coder2k.jpg". This will trigger a request to the backend. The route you're seeing here will then serve the desired image (if existing).
The image isn't served directly by the web server, because for CodeIgniter it seems to be the common way to *not* store uploaded files in a public directory, but rather provide access through a route.
An advantage to this could be, that it's possible to only allow logged-in users to access certain files. I don't know if we'll ever need this (since the filenames will be randomized), but this would be trivial to implement later on.

Reply:
got it.